cluster-provision/gocli/opts/cnao/cnao.go

```diff
@@ -54,7 +56,13 @@ func (o *cnaoOpt) Exec() error {
 					}
 
 					if o.multusEnabled {
-						re := regexp.MustCompile("(?m)[\r\n]+^.*multus.*$")
+						re := regexp.MustCompile("(?m)[\r\n]+^.*multus:.*$")
+						res := re.ReplaceAllString(string(yamlDoc), "")
+						yamlDoc = []byte(res)
+					}
```

Comment:
Please consider adding a second commit for removing this code path, so the only way to deploy Multus would be through CNAO.

Reply:
@orelmisan note that keeping this code path allow iterative  transition to Multus v4 (through CNAO only).
Removing this code path mean we need to transition all sig-network lanes to deploy Mutlus with CNAO.

Reply:
> @orelmisan note that keeping this code path allow iterative transition to Multus v4 (through CNAO only). Removing this code path mean we need to transition all sig-network lanes to deploy Mutlus with CNAO.

IIUC, if we will remove this code path, all network lanes will automatically have Multus v4.
The only lane that will break is the one with Multus v4 and DNC (and its periodic counterpart).
WDYT?

Reply:
This PR add flag for enable deploying some component, changing default behavior should be out of this PR scope IMO.

In addition, removing this code path alone wont do, the scripts that read the env vars and set the flags should be changed accordingly.
Then maybe it can work as you describe, and only because `KUBEVIRT_WITH_CNAO` is set all over the place (sig-net jobs).

Reply:
> This PR add flag for enable deploying some component, changing default behavior should be out of this PR scope IMO.

Fair enough.